v2/component_analyses.yaml

```diff
@@ -0,0 +1,169 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: "2"
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Dependency Analytics Server"
+security:
+- tokenAuth: []
+paths:
+  /component-analyses/{eco}/{name}/{ver}:
+    get:
+      tags:
+      - Scan Services
+      summary: Get security and other details about a software component and recommendations
+        to change to a more secured version
+      operationId: da_scanner.api_v2.component-analyses
+      parameters:
+      - name: eco
+        in: path
+        description: component ecosystem
+        required: true
+        schema:
+          type: string
+      - name: name
+        in: path
+        description: component name
+        required: true
+        schema:
+          type: string
+      - name: ver
+        in: path
+        description: component version
+        required: true
+        schema:
+          type: string
+      - name: security-vendor
+        in: header
+        description: Security Vendor
+        schema:
+          type: string
+          enum:
+          - snyk
+
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          content:
+            application/json:
+              schema:
+                items:
+                  oneOf:
+                    - $ref: '#/components/schemas/RegisteredUser'
+                    - $ref: '#/components/schemas/FreeUser'
+              examples:
+                RegisteredUser:
+                  value:
+                    recommendation:
+                      component_analyses:
+                        cve:
+                          - id: "CVE-2018-1002201"
+                            cvss: "1.2"
+
+                          - id: "CVE-2018-1002202"
+                            cvss: "1.3"
+                      recommended_version : 2
+                      message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+                      severity: "high"
+                      public_vulnerabilities_count: 2
+                      private_vulnerabilities_count: 1
+                      exploitable_vulnerabilities_count: 0
+                      vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+                FreeUser:
+                  value:
+                    recommendation:
+                      component_analyses:
+                        cve:
```

Comment:
I don't see any field named cve in both FreeUser and RegisteredUser

Reply:
Ack!